packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -99,6 +109,21 @@ export const ActionColumn = <T extends Record<string, unknown>>({
           menuItems={headerCheckBoxes}
           onHide={() => setIsOpen(false)}
           onShow={() => setIsOpen(true)}
+          {...(typeof columnManager !== 'boolean'
```

Comment:
What if instead of duplicating this `typeof` check in 3 places, you would create an object above and reuse it everywhere?
`const dropdownMenuProps = typeof columnManager !== 'boolean' ? columnManager.dropdownMenuProps : {};`

and then just
`...dropdownMenuProps.style`

Reply:
[DONE] Ohh right! Thanks for noticing that